pyicloud/base.py

```diff
@@ -72,17 +72,12 @@ def request(self, method, url, **kwargs):  # pylint: disable=arguments-differ
         if self.service.password_filter not in request_logger.filters:
             request_logger.addFilter(self.service.password_filter)
 
-        request_logger.debug(
-            "%s %s %s" % (
-                method,
-                url,
-                kwargs.get("data", "")
-            )
-        )
+        data = kwargs.get("data", "")
+        request_logger.debug(f"{method} {url} {data}")
```

Comment:
Logging is still usually done with `%s` argument and not f-strings. It has the benefit of not formatting the string until later. I suggest we keep the logging statements like this:

```py
logger.debug("Some logging message %s", parameter)
```

Reply:
The reason logging is still done using log parameters because it means the cost of converting the objects to string is only incurred when the logging is actually happening.